coalib/bears/requirements/NpmRequirement.py

```diff
@@ -49,7 +49,17 @@ def is_installed(self):
 
         :param return: True if dependency is installed, false otherwise.
         """
-        cmd = ['npm', 'show', self.package]
+        # we need to check the local and global install prefix
+        # local:
+        cmd = ['npm', 'list', self.package]
         if platform.system() == 'Windows':  # pragma: no cover
             cmd = ['cmd', '/c'] + cmd
-        return not call_without_output(cmd)
+        locally_installed = not call_without_output(cmd)
+
+        # global:
+        cmd = ['npm', 'list', '-g', self.package]
+        if platform.system() == 'Windows':  # pragma: no cover
+            cmd = ['cmd', '/c'] + cmd
+        globally_installed = not call_without_output(cmd)
```

Comment:
this is just a repetition of the previous block right ( the only difference is `cmd`) maybe... defining a function that does is a good idea so that things are not repeated... :sweat_smile: